test/util/bspNet/waits.ts

```diff
@@ -162,7 +164,8 @@ export const waitForBspStored = async (
         module: "fileSystem",
         method: "bspConfirmStoring",
         checkTxPool: true,
-        timeout: 300
+        timeout: 10000,
```

Comment:
This was not really necessary given that this function already iterates too. But it's fine anyway.